fw/src/mppt/local/pando.hpp

```diff
@@ -11,11 +11,12 @@
 #pragma once
 #include "../mppt.hpp"
 
-class PandO final : public MPPT {
+class PandO {
```

Comment:
Does this still work being used in the VoltageSweep global algorithm if it maintains the MPPT parent class?

I think it might be fine if both inherit MPPT since they can be decoupled from each other.